### DISCUSSION TITLE
Cloudflare Workers

### DISCUSSION THREAD

**Discussion by aligzl** [sanchezzzhak/node-device-detector#207] [Q&A]
Hi, Does this works on cloudflare workers?

Thanks

**Comment by sanchezzzhak** [comment]
Hi. It should work in some way.

CloudWorkers  or AWS Lamba Function, I don't have faith that they are good for complex things like a full-fledged application/service, it seems like it's just a developer's nightmare.
Cloud Workers 100k free request per day.  You can try it.
